src/js/live-tracker.js

```diff
@@ -47,6 +47,9 @@ class LiveTracker extends Component {
     this.reset_();
 
     this.on(this.player_, 'durationchange', this.handleDurationchange);
```

Comment:
this handler will get wrapped in an arrow function in this PR https://github.com/videojs/video.js/pull/7060